projects/rpx-xui-translation/src/lib/rpx-translation.service.ts

```diff
@@ -72,7 +72,7 @@ export class RpxTranslationService {
     ));
 
     return combineLatest(translatedComponents$).pipe(
-      map((values: string[]) => values.join(' ')) // Join the strings with a space or any separator
+      map((values: string[]) => values.join('')) // Join the strings with a space or any separator
```

Comment:
Fix the comment to say - join the strings without spaces - separators are included in the values